Compilation fails with VS2015

Not sure what's up. Looks like it just needs a sweep. At least this error is showing up:

```
error C3861: 'back_inserter': identifier not found
```

which indicates that `<iterator>` isn't being included.

Cc: @johnkit

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,6 +10,15 @@ if(NOT DEFINED CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_EXTENSIONS FALSE)
 endif()
 
+# This is needed for compatibility with ES_Superbuild
+if (NOT WIN32 AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  option(USE_CXX11_ABI_0 "Use the legacy GCC stdlib ABI" OFF)
+  mark_as_advanced(USE_CXX11_ABI_0)
+  if (USE_CXX11_ABI_0)
+    add_definitions("-D_GLIBCXX_USE_CXX11_ABI=0")
+  endif()
+endif ()
+
 
 # Add our Cmake directory to the module search path
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/CMake)

--- a/vtkMapMarkerSet.cxx
+++ b/vtkMapMarkerSet.cxx
@@ -51,6 +51,7 @@
 #include <cmath>
 #include <iostream>
 #include <iomanip>
+#include <iterator>  // std::back_inserter
 #include <vector>
 
 unsigned int vtkMapMarkerSet::NextMarkerHue = 0;

